### PR TITLE
Update-2022-09-26

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Update Backend](https://trello.com/c/K0jT9eOP/1023-update-backend)

File: datatypes/src/primitives/time_interval.rs [hunk 1]
```diff
@@ -61,6 +61,7 @@ impl TimeInterval {
     ///
     /// This constructor fails if `end` is before `start`
     ///
+    #[allow(clippy::trait_duplication_in_bounds)] // both `error::Error: From<A::Error> + From<B::Error>` is required
```
Comment (michaelmattig) [inline datatypes/src/primitives/time_interval.rs:64]: do we actually need to support the case where A!=B?

File: operators/src/processing/meteosat/reflectance.rs [hunk 2]
```diff
@@ -357,7 +357,9 @@ mod tests {
         let result = process_mock(ReflectanceParams::default(), Some(1), Some(1), true, None).await;
 
         assert!(result.is_ok());
-        println!("{:?}", result);
+
+        // println!("{:?}", result);
```
Comment (michaelmattig) [inline operators/src/processing/meteosat/reflectance.rs:361]: remove?

File: operators/src/processing/raster_vector_join/non_aggregated.rs [hunk 3]
```diff
@@ -828,7 +826,8 @@ mod tests {
         assert_eq!(result.len(), 1);
 
         let result = result.remove(0);
-        eprintln!("{}", result.to_geo_json());
+
+        // eprintln!("{}", result.to_geo_json());
```
Comment (michaelmattig) [inline operators/src/processing/raster_vector_join/non_aggregated.rs:830]: remove?

File: operators/src/processing/raster_vector_join/non_aggregated.rs [hunk 4]
```diff
@@ -988,7 +987,8 @@ mod tests {
...
```
Comment (michaelmattig) [inline operators/src/processing/raster_vector_join/non_aggregated.rs:991]: remove?

File: datatypes/src/primitives/time_interval.rs (hunk 1 above)
Comment (ChristianBeilschmidt) [reply datatypes/src/primitives/time_interval.rs:64]: There are rare occasions where we used

```Rust
TimeInterval::new(0, TimeInstance::MAX)
```

or 

```Rust
TimeInterval::new(TimeInstance::MIN, 1)
```

or in the constructor of `STRectangle`.

File: operators/src/processing/raster_vector_join/non_aggregated.rs (hunk 4 above)
Comment (ChristianBeilschmidt) [reply operators/src/processing/raster_vector_join/non_aggregated.rs]: [de9825b](https://github.com/geo-engine/geoengine/pull/631/commits/de9825b74fa2935b7d17d7e9d76724c0dac0f6b7)

File: operators/src/processing/raster_vector_join/non_aggregated.rs (hunk 3 above)
Comment (ChristianBeilschmidt) [reply operators/src/processing/raster_vector_join/non_aggregated.rs]: [de9825b](https://github.com/geo-engine/geoengine/pull/631/commits/de9825b74fa2935b7d17d7e9d76724c0dac0f6b7)

File: operators/src/processing/meteosat/reflectance.rs (hunk 2 above)
Comment (ChristianBeilschmidt) [reply operators/src/processing/meteosat/reflectance.rs]: [de9825b](https://github.com/geo-engine/geoengine/pull/631/commits/de9825b74fa2935b7d17d7e9d76724c0dac0f6b7)

**Review by michaelmattig — APPROVED** [review]

**Comment by michaelmattig** [comment]
bors merge

**Pull request merged**